README.md

```diff
@@ -36,6 +36,7 @@ $ npm install dynamo2es-lambda
 - **[errorHook]** - { Function(event, context, error) } - function to be called when error occurs; if hook is not provided, error is thrown
 - **[retryOptions]** - { Object } - retry configuration in case Elasticsearch indexing fails ([options description can be found here][promise-retry-url]) [is not retried by default]
 - **[transformRecordHook]** - { Function(record) } - optional function to perform custom data processing; accepts single record, must return processed object; useful for reshaping document before sending it to Elasticsearch
+- **[upsert]** - { Boolean } - Replaces index with update with partial document and uses upsert operation.
```

Comment:
As of curiosity: what's your use case? Why do you need to partially update a document? The original intent was to simply mirror DynamoDB in Elasticsearch.

Reply:
In my case I run a indexer that fills elasticsearch with related items ("fills the document"). When we hit the dynamo2es-lambda function and it indexes the item, it would remove the items that the indexer added.